scripts/generate-cocoa-bindings.ps1

```diff
@@ -28,12 +28,49 @@ if (!(Get-Command sharpie -ErrorAction SilentlyContinue))
 # Ensure Xamarin is installed (or sharpie won't produce expected output).
 if (!(Test-Path '/Library/Frameworks/Xamarin.iOS.framework/Versions/Current/lib/64bits/iOS/Xamarin.iOS.dll'))
 {
-    Write-Output 'Xamarin.iOS not found. Attempting to install via Homebrew.'
-    brew install --cask xamarin-ios
+    Write-Output 'Xamarin.iOS not found. Attempting to install manually.'
+
+    # Download Xamarin.iOS package
+    $packageName = 'xamarin.ios-16.4.0.23.pkg'
+    $directDownloadUrl = 'https://github.com/getsentry/sentry-dotnet/releases/download/1.0.0.0-xamarin-ios/Xamarin.iOS.16.4.0.23.pkg'
+    $downloadPath = "/tmp/$packageName"
+
+    Write-Output "Downloading Xamarin.iOS package..."
+    curl -L -o $downloadPath $directDownloadUrl
```

Comment:
should we verify the checksum before installing?

Reply:
Easy enough to add and makes things slightly more secure.